components/fxa-client/ios/FxAClient/PersistedFirefoxAccount.swift

```diff
@@ -3,6 +3,9 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import Foundation
+#if canImport(MozillaRustComponents)
```

Comment:
for my info, what will happen if we can't import it? If it will just break, but somewhere else, I wonder if we should remove the `canImport` so that it breaks here?

(I've no idea how this hangs together, so apologies if this makes absolutely zero sense :)

Reply:
totally understand the confusion! It exists simply because we're still in a hybrid state of consuming via external swift packages (Individual components: Nimbus, Viaduct, etc) vs having all the components together (current day). Ryan explains it well in this issue: https://github.com/mozilla/application-services/issues/4308

Relevant excerpt:
> The Nimbus swift bindings currently use various functions exposed by Glean, and broadly assume that they will be built together with Glean in a single combined Swift module. When we move to Swift Packages that’s no longer going to work, since consumers will want to depend on Glean as its own separate Swift package.